docker-compose.yaml

```
@@ -1,3 +1,9 @@
 services:
   LocalStack:
     image: ${DESKTOP_PLUGIN_IMAGE}
+    volumes:
+    - saved_config:/saved_config
+
+    
+volumes:
+  saved_config:
```

I would give this named volume (which will show up if you do a `docker volume ls`) a more descriptive name which indicates that this state is for the LocalStack app/extension, something like:

```suggestion
    volumes:
    - ls_app_config:/saved_config

volumes:
  ls_app_config:
```